src/items/alien-archives/irokiroi_oshirokiroi.json

```
@@ -954,14 +985,52 @@
         "damage": {
           "parts": []
         },
+        "damageNotes": "",
         "description": {
           "chat": "",
           "gmnotes": "",
           "short": "",
           "unidentified": "",
-          "value": "<p>When an oshirokiroi is killed, its gas bladder loses cohesion and explodes in a burst of high-frequency sound and toxic gases. Each creature within 20 feet of the oshirokiroi takes 6d6 sonic damage and is deafened for 1d4 rounds; with a successful DC 19 Fortitude save, a creature takes half the damage and negates the deafened effect. The area in a radius of 50 feet from the oshirokiroi becomes a dense cloud of toxic fumes that provide concealment. A creature that enters or starts its turn in the cloud takes 4d4 acid damage and is exposed to Osoran roil&nbsp;poison. The cloud can be dispersed in the same way as fog cloud and otherwise dissipates on its own in 1 minute.</p>"
+          "value": "<p>When an oshirokiroi is killed, its gas bladder loses cohesion and explodes in a burst of high-frequency sound and toxic gases. Each creature within 20 feet of the oshirokiroi takes 6d6 sonic damage and is deafened for 1d4 rounds; with a successful DC 19 Fortitude save, a creature takes half the damage and negates the deafened effect. The area in a radius of 50 feet from the oshirokiroi becomes a dense cloud of toxic fumes that provide concealment. A creature that enters or starts its turn in the cloud takes 4d4 acid damage and is exposed to Osoran roil&nbsp;@UUID[Compendium.sfrpg.rules.JaRMioj4isNp7r1P.JournalEntryPage.ZaS9H4rwaFIQTUBC]{poison}. The cloud can be dispersed in the same way as fog cloud and otherwise dissipates on its own in 1 minute.</p>"
```

```suggestion
          "value": "<p>When an oshirokiroi is killed, its gas bladder loses cohesion and explodes in a burst of high-frequency sound and toxic gases. Each creature within 20 feet of the oshirokiroi takes 6d6 sonic damage and is deafened for 1d4 rounds; with a successful DC 19 Fortitude save, a creature takes half the damage and negates the deafened effect. The area in a radius of 50 feet from the oshirokiroi becomes a dense cloud of toxic fumes that provide concealment. A creature that enters or starts its turn in the cloud takes 4d4 acid damage and is exposed to Osoran roil&nbsp;poison&nbsp;. The cloud can be dispersed in the same way as fog cloud and otherwise dissipates on its own in 1 minute.</p>"
```
This doesn't really need to be linked, wrapping terms that would need to linked in a non-breaking space doesn't flag an error